pkg/ssh/keymanager.go

```diff
@@ -164,13 +164,19 @@ func (km KeyManager) newCertRequired() bool {
 		level.Info(km.logger).Log("msg", "new certificate required: certificate is incorrect format")
 		return true
 	}
+
 	now := uint64(time.Now().Unix())
 
 	if now > cert.ValidBefore {
 		level.Info(km.logger).Log("msg", "new certificate required: certificate validity has expired")
 		return true
 	}
```

Comment:
This might be redundant, unless we want to keep it to differentiate between "has already expired" and "about to expire" events 🤔

Reply:
I think its worth keeping, just in case the Renew interval somehow becomes bigger than the validity window or something.